builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -679,7 +706,29 @@ func (b *backend) pathGetIssuerCRL(ctx context.Context, req *logical.Request, da
 		return nil, err
 	}
 
+	var certificate []byte
+	var contentType string
+
 	sc := b.makeStorageContext(ctx, req.Storage)
+	// tmp
+	responseHeaders := map[string][]string{}
+	if hasHeader(headerIfModifiedSince, req) {
+		before, err := sc.isIfModifiedSinceBeforeCRLConfigLastModified(req, responseHeaders)
+		if err != nil {
+			return nil, err // errorResponse, nil or nil, err?
+		}
+		if before {
+			return &logical.Response{
+				Data: map[string]interface{}{
+					logical.HTTPContentType: contentType,
+					logical.HTTPRawBody:     certificate,
+					logical.HTTPStatusCode:  304,
+				},
+				Headers: responseHeaders,
+			}, nil
+		}
+	}
+	// tmp
```

Comment:
You know this, but I'm just making sure I don't accidentally merge this with `// tmp` comments left in :-)